learnupon_exporter/management/commands/learnupon_export_enrollment_data.py

```diff
@@ -113,9 +113,9 @@ def export_enrollment_data(self, csv_file):
                 row["Enrollment Status"] = "completed"
```

Comment:
The client said "Completed Date missing for several records with Completed status". Could you remove this line?